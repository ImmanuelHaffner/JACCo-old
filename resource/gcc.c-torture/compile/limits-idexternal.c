int vLIM5a;
int vLIM5b;
