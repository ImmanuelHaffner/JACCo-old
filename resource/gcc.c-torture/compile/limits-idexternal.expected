0
limits-idexternal.c:1:1 keyword int
limits-idexternal.c:1:5 identifier vLIM5a
limits-idexternal.c:1:11 punctuator ;
limits-idexternal.c:2:1 keyword int
limits-idexternal.c:2:5 identifier vLIM5b
limits-idexternal.c:2:11 punctuator ;
